apps/tangle-dapp/data/restaking/useRestakingProfile.ts

```diff
@@ -27,10 +29,26 @@ const useRestakingProfile = () => {
     );
   }, [ledgerOpt]);
 
+  const totalRestaked = useMemo(
+    () =>
+      ledgerOpt?.isSome
+        ? // Dummy check to whether format the total restaked amount
+          // or not, as the local testnet is in wei but the live one is in unit
+          ledgerOpt.unwrap().total.toString().length > 10
+          ? +formatUnits(
+              ledgerOpt.unwrap().total.toBigInt(),
+              TANGLE_TOKEN_DECIMALS
+            )
+          : ledgerOpt.unwrap().total.toNumber()
```

Comment:
I think this may fail if the value is too big, instead it should be kept & dealt with as a BN